### PR TITLE
Notifications for the executed processes

### PR DESCRIPTION
**Pull request by Discookie**
Added notifications for adding CodeChecker log and analyze commands to the queue, and then executing them.
They are mainly displayed on the sidebar, and there's actions for killing the process or removing them from the queue.

### CODE REVIEW

File: src/backend/executor/process.ts [hunk 1]
```diff
@@ -270,20 +272,26 @@ export class ExecutorManager implements Disposable {
...
-    private updateStatus(status: ProcessStatus) {
+    private updateStatus([status, process]: [ProcessStatus, ScheduledProcess]) {
         switch (status) {
         case ProcessStatus.removed:
-            this._processStatusChange.fire(status);
+            this._processStatusChange.fire([status, process]);
             break;
         case ProcessStatus.running:
-            this._processStatusChange.fire(ProcessStatus.running);
+        case ProcessStatus.queued:
```
Comment (vodorok) [inline src/backend/executor/process.ts:281]: Is the above fallthrough on line 280 intentional?

**Review by vodorok — APPROVED** [review]
LGTM

File: src/backend/executor/process.ts (hunk 1 above)
Comment (Discookie) [reply src/backend/executor/process.ts:281]: It is, it's not really a fallthrough, just multiple values executing the same case with no extra code.

**Comment by vodorok** [comment]
Please resolve the yarn.lock confilcts then it can be merged.

**Pull request merged**